### ISSUE TITLE
Deploy fails due to identityservice incorrectly slices cert text

### ISSUE THREAD

**Issue by explosion78** [hyperledger/composer#1133]
## Context
Based on the composer-sample-applications-hlfv1, I've set up a project and created channel artefacts with my own crypto-config.yaml and configtx.yaml, in order to use my organisation names and so on. 

Can create and join channel, but while deploying the business network archive an error is returned. "Error received on pem.Decode of certificate"

## Expected Behavior

Should deploy successfully

## Actual Behavior

Error is returned with the message "Error received on pem.Decode of certificate"

## Possible Fix

It seems the error is returned while executing line 90 in identityservice.go in composer-runtime-hlfv1 (https://github.com/hyperledger/composer/blob/4f3f89bbd6c9de139d0daef91eefdbb9c7acff22/packages/composer-runtime-hlfv1/identityservice.go)

The byte array stored in creator variable is as follows:
[10,11,65,120,112,86,101,110,117,101,77,83,80,18,245,5,45,45,45,45,45,66,69,71,73,78,32,67,69,82,84,73,70,73,67,65,84,69,45,45,45,45,45,10,77,73,73,67,65,106,67,67,65,97,109,103,65,119,73,66,65,103,73,85,79,117,53,77,103,65,103,100,110,75,84,89,79,106,75,78,101,117,86,54,121,76,98,84,100,54,77,119,67,103,89,73,75,111,90,73,122,106,48,69,65,119,73,119,10,103,89,73,120,67,122,65,74,66,103,78,86,66,65,89,84,65,108,86,84,77,82,77,119,69,81,89,68,86,81,81,73,69,119,112,68,89,87,120,112,90,109,57,121,98,109,108,104,77,82,89,119,70,65,89,68,86,81,81,72,69,119,49,84,10,89,87,52,103,82,110,74,104,98,109,78,112,99,50,78,118,77,83,73,119,73,65,89,68,86,81,81,75,69,120,108,50,90,87,53,49,90,83,53,104,98,87,86,121,97,87,78,104,98,109,86,52,99,72,74,108,99,51,77,117,89,50,57,116,10,77,83,73,119,73,65,89,68,86,81,81,68,69,120,108,50,90,87,53,49,90,83,53,104,98,87,86,121,97,87,78,104,98,109,86,52,99,72,74,108,99,51,77,117,89,50,57,116,77,66,52,88,68,84,69,51,77,68,85,122,77,68,65,122,10,77,106,73,119,77,70,111,88,68,84,69,52,77,68,81,121,79,68,69,120,77,106,73,119,77,70,111,119,69,106,69,81,77,65,52,71,65,49,85,69,65,120,77,72,89,50,70,104,90,71,49,112,98,106,66,90,77,66,77,71,66,121,113,71,10,83,77,52,57,65,103,69,71,67,67,113,71,83,77,52,57,65,119,69,72,65,48,73,65,66,68,89,47,122,69,57,106,104,69,104,88,88,114,66,106,106,82,73,105,75,70,109,49,103,54,104,106,120,89,120,67,106,70,68,43,120,68,117,72,10,119,89,50,110,75,49,88,100,69,101,103,98,77,86,80,66,43,52,87,67,71,75,88,116,72,86,69,115,65,82,66,81,114,106,65,85,82,90,89,73,120,53,90,55,52,50,54,106,98,68,66,113,77,65,52,71,65,49,85,100,68,119,69,66,10,47,119,81,69,65,119,73,67,66,68,65,77,66,103,78,86,72,82,77,66,65,102,56,69,65,106,65,65,77,66,48,71,65,49,85,100,68,103,81,87,66,66,83,76,49,111,119,113,55,66,76,43,90,89,75,47,75,90,88,108,79,43,98,51,10,56,109,47,74,106,106,65,114,66,103,78,86,72,83,77,69,74,68,65,105,103,67,66,116,100,88,56,107,101,113,72,73,82,72,116,74,108,105,80,74,54,105,90,89,117,53,82,122,84,100,80,119,97,116,74,122,97,113,81,85,53,51,56,120,10,57,84,65,75,66,103,103,113,104,107,106,79,80,81,81,68,65,103,78,72,65,68,66,69,65,105,65,73,102,102,97,65,50,53,47,114,100,120,78,88,121,116,53,106,69,88,118,80,54,83,47,104,87,48,88,104,71,80,83,101,76,77,100,72,10,120,52,87,52,97,65,73,103,67,106,80,87,79,80,101,107,106,84,113,69,97,102,71,115,104,122,105,81,108,75,119,68,88,73,100,55,82,103,67,84,51,85,65,72,102,66,101,82,111,113,81,61,10,45,45,45,45,45,69,78,68,32,67,69,82,84,73,70,73,67,65,84,69,45,45,45,45,45,10]

When the line 82 in the same code is executed for the above array, the returned index position is set to 2. I don't really know Golang but possibly due to UTF-8 code point. Converting the array to string and finding index returns the same result. However using bytes.Index (https://golang.org/pkg/bytes/#Index) seems to work in this case.

## Steps to Reproduce

https://play.golang.org/p/Cfb5n3lF14

```
package main

import (
	"bytes"
	//"strings"
	"fmt"
)

func main() {
	buf := []byte{10,11,65,120,112,86,101,110,117,101,77,83,80,18,245,5,45,45,45,45,45,66,69,71,73,78,32,67,69,82,84,73,70,73,67,65,84,69,45,45,45,45,45,10,77,73,73,67,65,106,67,67,65,97,109,103,65,119,73,66,65,103,73,85,79,117,53,77,103,65,103,100,110,75,84,89,79,106,75,78,101,117,86,54,121,76,98,84,100,54,77,119,67,103,89,73,75,111,90,73,122,106,48,69,65,119,73,119,10,103,89,73,120,67,122,65,74,66,103,78,86,66,65,89,84,65,108,86,84,77,82,77,119,69,81,89,68,86,81,81,73,69,119,112,68,89,87,120,112,90,109,57,121,98,109,108,104,77,82,89,119,70,65,89,68,86,81,81,72,69,119,49,84,10,89,87,52,103,82,110,74,104,98,109,78,112,99,50,78,118,77,83,73,119,73,65,89,68,86,81,81,75,69,120,108,50,90,87,53,49,90,83,53,104,98,87,86,121,97,87,78,104,98,109,86,52,99,72,74,108,99,51,77,117,89,50,57,116,10,77,83,73,119,73,65,89,68,86,81,81,68,69,120,108,50,90,87,53,49,90,83,53,104,98,87,86,121,97,87,78,104,98,109,86,52,99,72,74,108,99,51,77,117,89,50,57,116,77,66,52,88,68,84,69,51,77,68,85,122,77,68,65,122,10,77,106,73,119,77,70,111,88,68,84,69,52,77,68,81,121,79,68,69,120,77,106,73,119,77,70,111,119,69,106,69,81,77,65,52,71,65,49,85,69,65,120,77,72,89,50,70,104,90,71,49,112,98,106,66,90,77,66,77,71,66,121,113,71,10,83,77,52,57,65,103,69,71,67,67,113,71,83,77,52,57,65,119,69,72,65,48,73,65,66,68,89,47,122,69,57,106,104,69,104,88,88,114,66,106,106,82,73,105,75,70,109,49,103,54,104,106,120,89,120,67,106,70,68,43,120,68,117,72,10,119,89,50,110,75,49,88,100,69,101,103,98,77,86,80,66,43,52,87,67,71,75,88,116,72,86,69,115,65,82,66,81,114,106,65,85,82,90,89,73,120,53,90,55,52,50,54,106,98,68,66,113,77,65,52,71,65,49,85,100,68,119,69,66,10,47,119,81,69,65,119,73,67,66,68,65,77,66,103,78,86,72,82,77,66,65,102,56,69,65,106,65,65,77,66,48,71,65,49,85,100,68,103,81,87,66,66,83,76,49,111,119,113,55,66,76,43,90,89,75,47,75,90,88,108,79,43,98,51,10,56,109,47,74,106,106,65,114,66,103,78,86,72,83,77,69,74,68,65,105,103,67,66,116,100,88,56,107,101,113,72,73,82,72,116,74,108,105,80,74,54,105,90,89,117,53,82,122,84,100,80,119,97,116,74,122,97,113,81,85,53,51,56,120,10,57,84,65,75,66,103,103,113,104,107,106,79,80,81,81,68,65,103,78,72,65,68,66,69,65,105,65,73,102,102,97,65,50,53,47,114,100,120,78,88,121,116,53,106,69,88,118,80,54,83,47,104,87,48,88,104,71,80,83,101,76,77,100,72,10,120,52,87,52,97,65,73,103,67,106,80,87,79,80,101,107,106,84,113,69,97,102,71,115,104,122,105,81,108,75,119,68,88,73,100,55,82,103,67,84,51,85,65,72,102,66,101,82,111,113,81,61,10,45,45,45,45,45,69,78,68,32,67,69,82,84,73,70,73,67,65,84,69,45,45,45,45,45,10}
	
	startWrong := bytes.IndexAny(buf, "----BEGIN CERTIFICATE-----")
	startCorrect := bytes.Index(buf,[]byte{45,45,45,45,45,66,69,71,73,78,32,67,69,82,84,73,70,73,67,65,84,69,45,45,45,45,45})

	fmt.Println("Wrong Index: ", startWrong)
	fmt.Println("Correct Index: ", startCorrect)
	
	certTextWrong := buf[startWrong:]
	certTextCorrect := buf[startCorrect:]
	
	fmt.Println("Wrong PEM:\n", string(certTextWrong))
	fmt.Println("Correct PEM:\n", string(certTextCorrect))
}
```

## Existing issues

- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-composer)
- [ ] [GitHub Issues](https://github.com/hyperledger/composer/issues)
- [ ] [Rocket Chat history](https://chat.hyperledger.org/channel/composer)

## Your Environment

* Version used: 0.7.3
* Environment name and version (e.g. Chrome 39, node.js 5.4):
* Operating System and version (desktop or mobile):
* Link to your project:

**Comment by davidkel** [comment]
This is now included in the support for fabric 1.0.0-beta

**Comment by davidkel** [comment]
now in master
